collectd-jenkins/README.md

```diff
@@ -0,0 +1,229 @@
+# ![](./img/integrations_jenkins.png) Jenkins
+
+_This directory consolidates all the metadata associated with the jenkins plugin for collectd. The relevant code for the plugin can be found [here](https://github.com/signalfx/collectd-jenkins)_
```

Comment:
You can remove the underscores (avoiding italics) to be consistent with the new formatting